pkg/cloudprovider/aws/launchtemplate/bootstrap/bottlerocket.go

```diff
@@ -0,0 +1,73 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/pelletier/go-toml/v2"
+)
+
+type Bottlerocket struct {
```

Comment:
Thoughts on bundling this implementation with the launch template implementation? It already knows how to translate to a bootstrap options, so it could just implement the Script() interface itself.